openssl3/install-sfw

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/ksh93
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright 2010 Sun Microsystems, Inc.  All rights reserved.
+# Use is subject to license terms.
+#
+# Copyright 2021 Joyent, Inc.
+#
+
+set -o errexit
+
+LIBDIR=${DESTDIR}/lib
+BINDIR=${DESTDIR}/usr/bin
+USRLIBDIR=${DESTDIR}/usr/lib
+INCDIR=${DESTDIR}/usr/include
+
+mkdir -p $LIBDIR
+mkdir -p $BINDIR
+mkdir -p $USRLIBDIR
+mkdir -p $INCDIR
```

Comment:
Technically these should be quoted I believe.  I'm not a bash expert but did you run this through shell check tool?